jaffle_shop/models/staging/src_seed/_models.yml

```diff
@@ -1,13 +1,27 @@
 version: 2
 
 models:
-  - name: stg_customers
+  - name: stg_customers_pii
+    meta:
+      sensitive: true
     columns:
       - name: customer_id
         tests:
           - unique
           - not_null
+      - name: first_name
+        meta:
+          sensitive: true
+      - name: last_name
+        meta:
+          sensitive: true
 
+  - name: stg_customers
```

Comment:
in staging tables after sensitive tables we should also have test that makes sure hash column exists:

>     columns:
>       - name: address_line1_hash
>         tests:
>           - dbt_expectations.expect_column_to_exist 

This is because a bug with the hash macro that occasionally creates sensitive data columns instead of their hash